content/docs/gto/get-started-dvc.md

```diff
@@ -0,0 +1,115 @@
+# Get Started DVC
+
+To leverage concepts of Model and Data Registries in a more explicit way, you
+can denote the `type` of each output. This will let you browse models and data
+separately, address them by `name` in `dvc get`, and eventually, see them in DVC
+Studio.
+
+Let's start with marking an artifact as data or model.
```

Comment:
Why? I assumed Data Registry should show `type: data`.

Reply:
It carries 0 information though, right?
"data of type data", similar to "artifact of type artifact" means the same as not defining type at all. it's the most general thing there is (data even more than artifact maybe?), just super abstract. doubt users will use it that way

Reply:
Good point. Maybe it's `dataset` instead of `data` then? Or anyways, if after subtracting `plots`, `metrics` and `models` everything that's left (among DVC PL inputs and outputs) is `dataset`, then I guess it's redundant?

Reply:
My 2cs. Data is not abstract for me (it's different vs model in my perception). But in DVC it's not needed. Any `out` w/o a specified type can be considered data. 

I would personally try to simplify all of this - no multiple types initially. only models. We are prematurely generalizing this I think.

Reply:
I think of the terms `Data Registry`, `Model Registry`, and `Artifact Registry`. I like having the type track to those names, so I like `data`.

Would love to keep the idea of data registry support around, haven't thought a lot about using `gto` for that, but certainly have thought about models and binaries.

I would love to use/try `gto` as an `Artifact Registry` for build artifacts - specifically compiled binaries.  It might also be interesting to use as a `Container Registry` - there are lots of solutions in this space like [Jfrog Artifactory](https://jfrog.com/artifactory/), [cloudsmith](https://cloudsmith.com/), [GCP](https://cloud.google.com/build/docs/building/store-artifacts-in-cloud-storage).  If you wanted to be really fancy you could support some of those offerings as backends.

With all of that said I think prioritizing the `Model Registry` use case makes a lot of sense, a `Data Registry` would be my next priority.  The solution seems like it may be general enough to support an `Artifact Registry` and `Container Registry`, might be worth doing some thinking about it and if it does make sense and there are advantages to keeping that in `gto` then keep that use case in mind while making the `Model Registry` experience awesome.

Reply:
Trying to think of advantages of having `gto` also be a `Container Registry` and remembered that `mlem`  can deploy docker containers, it would be nice to have the versioning of those container artifacts in `gto` and be able to reference with `gto` syntax.

Reply:
If I might chip in as a user - I absolutely would like GTO to be used also to build a data(set) registry (together with DVC) and possibly a combined data(set) and model registry.

In fact, we intend to use GTO and DVC to build a dataset registry for one (fairly large) client in the coming weeks. By the way, while stuff like MLFlow is a viable alternative to the GTO+MLEM-based model registry (it does not have all of its features but has some others), I don't know any open source alternative to a GTO+DVC-based dataset registry (and general I've yet to see data versioning done better than with DVC)...which means it is a very good selling point IMO.

I like the idea of viewing GTO as a tool to build pretty much any artifact registry, though models and datasets are the most likely use-cases.